Microsoft.Toolkit.Uwp.UI.Controls/OrbitView/OrbitView.cs

```diff
@@ -0,0 +1,687 @@
+    /// </summary>
+    [TemplatePart(Name = "AnchorCanvas", Type = typeof(Canvas))]
+    [TemplatePart(Name = "OrbitGrid", Type = typeof(Grid))]
+    [TemplatePart(Name = "CenterContent", Type = typeof(ContentPresenter))]
+    public sealed class OrbitView : ItemsControl
+    {
+        private const double _animationDuration = 200;
+
+        private OrbitViewPanel _panel;
+        private Grid _orbitsContainer;
+        private Canvas _anchorCanvas;
+        private ContentPresenter _centerContent;
+        private Compositor _compositor;
+
+        private Dictionary<object, Ellipse> _orbits;
+        private Dictionary<object, Line> _anchors;
+
+        /// <summary>
+        /// Raised when an item has been clicked or activated with keyboard/controller 
+        /// </summary>
+        public event EventHandler<OrbitViewItemClickedEventArgs> ItemInvoked;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OrbitView"/> class.
+        /// Creates a new instance of <see cref="OrbitView"/>
+        /// </summary>
+        public OrbitView()
+        {
+            DefaultStyleKey = typeof(OrbitView);
+
+            IsTabStop = false;
+            TabNavigation = KeyboardNavigationMode.Once;
+            KeyDown += OrbitView_KeyDown;
+            _orbits = new Dictionary<object, Ellipse>();
+            _anchors = new Dictionary<object, Line>();
+
+            if (Windows.ApplicationModel.DesignMode.DesignModeEnabled)
+            {
+                var items = new List<OrbitViewItem>();
+                items.Add(new OrbitViewItem() { Distance = 0.1, Diameter = 0.5, Label = "test" });
+                items.Add(new OrbitViewItem() { Distance = 0.1, Diameter = 0.5, Label = "test" });
+                items.Add(new OrbitViewItem() { Distance = 0.1, Diameter = 0.5, Label = "test" });
+                ItemsSource = items;
+            }
+        }
+
+        protected override void OnApplyTemplate()
+        {
+            _centerContent = (ContentPresenter)GetTemplateChild("CenterContent");
+            if (_centerContent == null)
+            {
+                return;
+            }
+
+            _compositor = ElementCompositionPreview.GetElementVisual(this).Compositor;
+
+            base.OnApplyTemplate();
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether orbits are enabled or not.
+        /// </summary>
+        public bool OrbitsEnabled
+        {
+            get { return (bool)GetValue(OrbitsEnabledProperty); }
+            set { SetValue(OrbitsEnabledProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="OrbitsEnabledProperty"/> property
+        /// </summary>
+        public static readonly DependencyProperty OrbitsEnabledProperty =
+            DependencyProperty.Register("OrbitsEnabled", typeof(bool), typeof(OrbitView), new PropertyMetadata(false, OnOrbitsEnabledChanged));
```

Comment:
"OrbitsEnabled" should be nameof(OrbitsEnabled).
Please apply for rest of properties.